doc/json-rpc.md

```diff
@@ -0,0 +1,148 @@
+Solana JSON RPC API
+===
+
+Solana nodes accept HTTP requests using the [JSON-RPC 2.0](https://www.jsonrpc.org/specification) specification.
+
+To interact with a Solana node inside a JavaScript application, use the [solana-web3.js](https://github.com/solana-labs/solana-web3.js) library, which gives a convenient interface for the RPC methods.
+
+RPC Endpoint
+---
+
+**Default port:** 8899  
+eg. http://localhost:8899, 192.168.1.88:8899
+
+Methods
+---
+
+* [confirmTransaction](#confirmtransaction)
+* [getAddress](#getaddress)
+* [getBalance](#getbalance)
+* [getLastId](#getlastid)
+* [getTransactionCount](#gettransactioncount)
+* [sendTransaction](#sendtransaction)
+
+Request Formatting
+---
+
+To make a JSON-RPC request, send an HTTP POST request with a `Content-Type: application/json` header. The JSON request data should contain 4 fields:
+
+* `jsonrpc`, set to `"2.0"`
+* `id`, a unique client-generated identifying integer
+* `method`, a string containing the method to be invoked
+* `params`, a JSON array of ordered parameter values
+
+Example using curl:
+```bash
+curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "method":"getBalance", "params":["83astBRguLMdt2h5U1Tpdq5tjFoJ6noeGwaY3mDLVcri"]}' 192.168.1.88:8899
+```
+
+The response output will be a JSON object with the following fields:
+
+* `jsonrpc`, matching the request specification
+* `id`, matching the request identifier
+* `result`, requested data or success confirmation
+
+Requests can be sent in batches by sending an array of JSON-RPC request objects as the data for a single POST.
+
+JSON RPC API Reference
+---
+
+### confirmTransaction
+Returns a transaction receipt
+
+##### Parameters:
+* `string` - SIGNATURE of transaction to confirm, as base-58 encoded string
```

Comment:
SIGNATURE/PUBKEY/HASH - there's a little bit of yelling in this doc.  Perhaps we define these words at the top of the doc then use them as proper nouns throughout the rest of the doc:  Signature/PublicKey/Hash

Reply:
Like it.